scripts/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+if [ -z "$1" ]; then 
+    BUILD_TYPE=Release
+else 
+    BUILD_TYPE="$1"
+fi
+
+cd $(git rev-parse --show-toplevel)
```

Comment:
Please use things like `$(dirname "$0")` - we cannot assume that this is a git repository, for example the source tarballs don't have one.